packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -182,10 +184,19 @@ export default defineComponent({
       return props.breadcrumbs.length <= 1
     })
 
+    const routeMetaTitle = useRouteMeta('title')
+    const pageTitle = computed(() => {
+      if (unref(routeMetaTitle)) {
+        return $gettext(unref(routeMetaTitle))
+      }
+      return props.space.name
```

Comment:
`space` could be `null` theoretically, it would probably make sense to catch that, what do you think?